apps/registry/pages/interview.js

```diff
@@ -134,9 +130,7 @@ export default function Talk() {
   const [text, setText] = useState('');
   const [reply, setReply] = useState('');
   const [replying, setReplying] = useState(null);
-  const [loading, setLoading] = useState(false);
-  const [showAbout, setShowAbout] = useState(false);
-  const [error, setError] = useState(null);
```

Comment:
A lot of this stuff was never read. Since I fixed eslint, it now complains. Removed it (alt + shift + o in vscode on linux)